zulip_botserver/zulip_botserver/server.py

```diff
@@ -28,6 +29,32 @@ def read_config_section(parser: configparser.ConfigParser, section: str) -> Dict
     }
     return section_info
 
+def read_config_from_env_vars(bot_name: Optional[str] = None) -> Dict[str, Dict[str, str]]:
+    bots_config = {}  # type: Dict[str, Dict[str, str]]
+    env_config = OrderedDict()   # type: Dict[str, Dict[str, str]]
+    json_config = os.environ.get('ZULIP_BOTSERVER_CONFIG')   # type: Optional[str]
```

Comment:
mypy can deduce the type of this, so we don't need the `# type:` comment here -- and I don't think it it adds much.